tools/docker-compose/bootstrap_development.sh

```diff
@@ -53,5 +53,5 @@ fi
 
 # Create resource entries when using Minikube
 if [[ -n "$MINIKUBE_CONTAINER_GROUP" ]]; then
-    awx-manage shell < /awx_devel/tools/docker-compose-minikube/_sources/bootstrap_minikube.py
+    awx-manage shell < /awx_devel/tools/docker-compose-minikube/"${SOURCES:-_sources}"/bootstrap_minikube.py
```

Comment:
I'm not sure what this is doing, I don't see how it can be correct. Won't the sources directory be like `tools/docker-compose/_sources/`?

Reply:
You're right. I think I missed that one. I should test this more with the minikube stuff. :/ I think I know of a way to fix it for this PR that won't effect anything else. Let me try something.

Reply:
PTAL. I think this will actually do the trick. Fixed up the `docker-compose-sources` rule in the `Makefile` to pass in `$(SOURCES)` to the minikube playbook. The minikube role didn't need any changes as it was using its `sources_dest` var throughout. I haven't tested it yet, but I'm about to.